tests/integration/test_cli.py

```diff
@@ -48,10 +49,7 @@
     ],
 )
 def test_multipart_download(
-    file_size,
-    part_size,
-    s3_fixture: S3Fixture,  # noqa F811
-    tmp_path,
+    file_size, part_size, s3_fixture: S3Fixture, tmp_path, max_retries: int  # noqa F811
```

Comment:
I think we shouldn't use noqa broadly, but provide the funciton arguments one line each and only use noqa statements where necessary
```suggestion
    file_size, 
    part_size, 
    s3_fixture: S3Fixture,  # noqa F811
    tmp_path, 
    max_retries: int  # noqa F811
```

Reply:
Thanks, missed that as it got autoformatted to this form, before I added the comment